https://issues.redhat.com/browse/ACM-11779 (#6616)
open-cluster-management/rhacm-docs: governance/config_policy_ctrl.adoc

| `spec.object-templates[].complianceType`
| Required

> a| Used to define the desired state of the Kubernetes object on the managed clusters. You must use one of the following verbs as the parameter value:

Reviewer: optional, increase user focus: 

```suggestion
a| Use it to define the desired state of the Kubernetes object on your managed clusters. Use one of the following verbs as the parameter value:
```

Author: ```suggestion
a| Use this parameter to define the desired state of the Kubernetes object on your managed clusters. Use one of the following verbs as the parameter value:
```

- `mustonlyhave`: Indicates that an object must exist with the exact fields and values as defined in the `objectDefinition`.

- `musthave`: Indicates an object must exist with the same fields as specified in the `objectDefinition`. Any existing fields on the object that are not specified in the `object-template` are ignored. In general, array values are appended. The exception for the array to be patched is when the item contains a `name` key with a value that matches an existing item. Use a fully defined `objectDefinition` using the `mustonlyhave` compliance type, if you want to replace the array.

- `mustnothave`: Indicates that an object with the same fields as specified in the `objectDefinition` cannot exist.

| `spec.object-templates[].metadataComplianceType`
| Optional
| Overrides `spec.object-templates[].complianceType` when comparing the manifest's metadata section to objects on the cluster ("musthave", "mustonlyhave"). Default is unset to not override `complianceType` for metadata.

| `spec.object-templates[].recordDiff`
| Optional

> a| Specifies if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. The following options are supported:

Reviewer: optional, again with the user focus: 

```suggestion
a| Use it to specify if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. Choose from the following supported options:
```

Author: ```suggestion
a| Use this parameter to specify if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. The following options are supported:
```
